From: Simon McVittie <smcv@debian.org>
Date: Sat, 4 Nov 2017 14:57:44 +0000
Subject: build: Put third-party macros in m4/

aclocal can't install third-party macros, as required for the recommended
invocation of intltoolize from
https://wiki.gnome.org/Projects/GnomeCommon/Migration, without somewhere
to put them.

Signed-off-by: Simon McVittie <smcv@debian.org>
Bug-Debian: https://bugs.debian.org/829972
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=789906
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index feb592c..1db8499 100644
--- a/configure.ac
+++ b/configure.ac
@@ -4,6 +4,7 @@ AC_INIT([d-feet],
 	[http://bugzilla.gnome.org/enter_bug.cgi?product=d-feet],
         [d-feet],
         [https://wiki.gnome.org/Apps/DFeet])
+AC_CONFIG_MACRO_DIR([m4])
 
 AM_INIT_AUTOMAKE([1.11 -Wall foreign no-define tar-ustar no-dist-gzip dist-xz])
 
